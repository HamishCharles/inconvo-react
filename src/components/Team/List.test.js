import React from 'react';
import TeamList from './List';
import renderer from 'react-test-renderer';


test('Renders team list correctly', () => {
  const component = renderer.create(
    <TeamList filteredTeams={[{name: 'Blah', img: 'blah.jpg'},{name: 'Rovers', img: 'rovers.jpg'}]} teamClick={function(){}}></TeamList>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});