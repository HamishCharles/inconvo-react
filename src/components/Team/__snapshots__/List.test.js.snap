// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders team list correctly 1`] = `
<div
  className="team-list"
>
  <div
    className="team"
    onClick={[Function]}
  >
    <img
      alt="Blah logo"
      onAbort={[Function]}
      onError={[Function]}
      src="blah.jpg"
    />
    <div
      className="overlay"
    >
      <span
        className="team-name"
      >
        Blah
      </span>
    </div>
  </div>
  <div
    className="team"
    onClick={[Function]}
  >
    <img
      alt="Rovers logo"
      onAbort={[Function]}
      onError={[Function]}
      src="rovers.jpg"
    />
    <div
      className="overlay"
    >
      <span
        className="team-name"
      >
        Rovers
      </span>
    </div>
  </div>
</div>
`;
